server.js

```diff
@@ -85,9 +85,7 @@ module.exports.initExpress = function () {
   app.set('trust proxy', config.trustProxy);
 
   // If we're set up with Sentry, use its middleware to record requests.
-  if (config.sentryDsn) {
-    app.use(Sentry.Handlers.requestHandler());
-  }
+  app.use(Sentry.Handlers.requestHandler());
```

Comment:
The comment implies we only load this middleware if we are using Sentry, but the code now seems to load it unconditionally?

Reply:
Good catch, let me clean up the comments.